Nef_3/include/CGAL/Nef_polyhedron_3.h

```diff
@@ -53,6 +53,8 @@
 #include <CGAL/Nef_3/shell_to_nef_3.h>
 #include <CGAL/Polyhedron_incremental_builder_3.h>
 #include <CGAL/Polyhedron_3.h>
+#include <CGAL/boost/graph/graph_traits_Polyhedron_3.h>
+#include <CGAL/boost/graph/properties_Polyhedron_3.h>
```

Comment:
That's right: `<CGAL/boost/graph/convert_nef_polyhedron_to_polygon_mesh.h>` uses the BGL API and is agnostic of Polyhedron_3.

Thanks for the answer.